plugins/gnostic-plugin-request/main.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// gnostic-plugin-request is a development tool that captures and optionally
+// displays the contents of the gnostic plugin interface.
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	openapiv2 "github.com/googleapis/gnostic/OpenAPIv2"
+	openapiv3 "github.com/googleapis/gnostic/OpenAPIv3"
+	plugins "github.com/googleapis/gnostic/plugins"
+	surface "github.com/googleapis/gnostic/surface"
+)
+
+// Record an error, then serialize and return a response.
```

Comment:
Nit (feel free to ignore): In case you are concerned with GoDoc format, function level comments needs to start with the function name (even unexported methods).

Some resources:
* https://blog.golang.org/godoc-documenting-go-code
* https://www.digitalocean.com/community/tutorials/how-to-write-comments-in-go

Reply:
Thanks! On examination, I found that those functions weren't being used (copy-and-paste leftovers), so I removed them.